.github/workflows/tests-and-docs.yml

```diff
@@ -57,7 +57,11 @@ jobs:
         # retrieve latest model for img2mml component
         curl -L https://artifacts.askem.lum.ai/skema/img2mml/models/cnn_xfmer_arxiv_im2mml_with_fonts_boldface_best.pt > skema/img2mml/trained_models/cnn_xfmer_arxiv_im2mml_with_fonts_boldface_best.pt
         pip install ".[all]"
-
+    # Install tree-sitter parser (for Python component unit tests)
+    - name: Install tree-sitter parsers
+      working-directory: .
+      run: python skema/program_analysis/tree_sitter_parsers/build_parsers.py --c --cpp --fortran --python --matlab --r
```

Comment:
Thanks, @vincentraymond-ua !  Can you also update the dev environment documentation?  https://ml4ai.github.io/skema/dev/env/#configuring-your-dev-environment

Any idea why the matlab and fortran tests were passing?